### DISCUSSION TITLE
DiscussionForum: Topic creation is failing

### DISCUSSION THREAD

**Discussion by vinukumar-vs** [Sunbird-Lern/Community#116] [Issues] [answered]
After login to Portal, we are trying to create the topic for the forum attached to the group. But we are getting below error
<img width="1792" alt="Screenshot 2023-05-13 at 2 49 40 PM" src="https://github.com/Sunbird-Lern/Community/assets/4496096/cd91a52e-5c2e-4cc9-9b4c-7f2fcb7f49fd">

Please find the below is the log 
> [32m[2023-05-13 09:14:58.005] [INFO] api - [39m{"eid":"LOG","ets":1518460198146,"ver":"3.0","mid":"LOG:69e9ca45-c7e2-4a94-af50-50a4ff854cc9","actor":{"id":"discussion-forum-middleware","type":"API"},"context":{"env":"/discussion/v2/topics","did":"99d611118eeabdb62bdda0abc8512c21","sid":"B3jVrI1U36_L0bwp_S7KxW9jiW8QXJT4","pdata":{"id":"org.sunbird.discussion-forum-middleware","pid":"","ver":"1.0"},"cdata":[]},"edata":{"type":"log","level":"WARN","requestid":"","message":"You are not authorized to access /discussion/v2/topics","msgid":"e3d4070361e44e7c2d99658e61e81a57"}}

**Comment by reshmi-nair** [comment, marked as answer]
@vinukumar-vs Currently discussion-mw supports only one pod, if more than one pod is used, connection exceptions might occur. Also session expiry issue will occur, if the master token created from nodebb admin console is not added in configuration.
Please refer the link for master token creation : [master token](https://lern.sunbird.org/use/developer-installation/discussion-forum/installation-guide/nodebb-setup) and [token update](https://lern.sunbird.org/learn/product-and-developer-guide/discussion-forum/configurations/discussion-mw-and-nodebb-system-config)
